scout/server/blueprints/login/views.py

```diff
@@ -66,20 +72,19 @@ def login():
             flash("User not authorized by LDAP server", "warning")
             return redirect(url_for("public.index"))
 
-    if current_app.config.get("GOOGLE"):
+    elif current_app.config.get("GOOGLE"):
```

Comment:
They become mutually exclusive. I think it gives more control to the thing

Reply:
Yes: I had to reread because I thought we could rearrange and have the common returns first, but I agree with this.